README.md

```diff
@@ -1,6 +1,6 @@
-<img src="https://raw.githubusercontent.com/Workflomics/.github/main/WORKFLOMICS_logo_small.jpg" alt="logo" width=60%/>
+<img src="https://github.com/Workflomics/.github/blob/main/WORKFLOMICS_logo.png" alt="logo" width=60%/>
```

Comment:
Is there a reason why this isn't just a relative reference to a logo image inside the repo?

Reply:
Because the logo within the repo doesn't have a background, and therefore, does not work well with a dark GitHub theme. I created the [new logo](https://github.com/Workflomics/.github/blob/main/WORKFLOMICS_logo.png) to look nice when using a dark theme.

Reply:
It looks like this 
<img width="1157" alt="Screenshot 2024-05-21 at 21 50 16" src="https://github.com/Workflomics/workflomics-frontend/assets/11068408/e294f63b-fd4d-47ba-82f5-d0a09de2aca6">